pkg/templates/charts/toggle/managed-serviceaccount/templates/addontemplate.yaml

```diff
@@ -0,0 +1,135 @@
+apiVersion: addon.open-cluster-management.io/v1alpha1
+kind: AddOnTemplate
+metadata:
+  name: managed-serviceaccount
```

Comment:
should you have the version in the name?

Reply:
`managed-serviceaccount-2.4` ?